engine/src/srvoutput.cpp

```diff
@@ -509,11 +509,15 @@ bool MCServerSetCookie(MCStringRef p_name, MCStringRef p_value, uint32_t p_expir
 	
 	if (t_success)
 	{
-        MCservercgicookies[t_index].name = MCStringIsEmpty(p_name) ? strdup("") : strdup(MCStringGetCString(p_name));
-		MCservercgicookies[t_index].value = strdup(MCStringGetCString(*t_encoded));
-        MCservercgicookies[t_index].path = MCStringIsEmpty(p_domain) ? strdup("") : strdup(MCStringGetCString(p_path));
-        MCservercgicookies[t_index].domain = MCStringIsEmpty(p_domain) ? strdup("") : strdup(MCStringGetCString(p_domain));
+		// SN-2015-05-14: [[ MCStringGetCString Removal ]] Use ConvertToCString
+		t_success = MCStringConvertToCString(p_name, MCservercgicookies[t_index].name)
```

Comment:
Would it not be better to upgrade the MCservercgicookies arrays to use MCStringRef's?